docusaurus.config.js

```diff
@@ -25,6 +25,53 @@ module.exports = {
   favicon: 'img/meta/favicon-96x96.png',
   organizationName: 'ionic-team',
   projectName: 'ionic-docs',
+  presets: [
```

Comment:
The main body alignment is incorrect on smaller viewports:

| production | branch |
| - | - |
| ![image](https://github.com/ionic-team/ionic-docs/assets/2721089/e77bfab9-186a-4f62-856a-eb95752d4bbe) | ![image](https://github.com/ionic-team/ionic-docs/assets/2721089/e826cf1a-329e-4c0e-8990-5fe7e9d9db84) |